### ISSUE TITLE
Following the "Browser (no bundling)" in the README leads to missing "global".

### ISSUE THREAD

**Issue by VorpalBlade** [mongodb/js-bson#330]
Following the instruction in the section "Browser (no bundling)" doesn't work.

On line 3726, "global" is used, which is apparently not defined in browsers.

In Firefox:
```
ReferenceError: global is not defined                                  bson.bundle.js:3726:6
    BSON https://automower2/js/bson.bundle.js:3726
    <anonymous> https://automower2/js/bson.bundle.js:9112
```
Chrome has a similar error.

**Comment by captaincaius** [comment]
I'm not sure if this deserves a separate issue because it might be related.  The "with bundling" section also has the same problem, and it seems the offending lines of code come from elsewhere:

```javascript
if (typeof global.setTimeout === 'function') {
  cachedSetTimeout = setTimeout;
}

if (typeof global.clearTimeout === 'function') {
  cachedClearTimeout = clearTimeout;
}
```

**Comment by captaincaius** [comment]
one more...

var performance = global.performance || {};

**Comment by daprahamian** [comment]
Filed https://jira.mongodb.org/browse/NODE-2240 to track

**Comment by nbbeeken** [comment]
Just for reference we use JIRA to track our issues and prefer that issues are submitted there so we can better prioritize them.
Thanks for submitting! resolved in PR #404
